lib/vulcan.js

```diff
@@ -51,7 +51,7 @@ function buildLoader(abspath, excludes) {
 var Vulcan = function Vulcan(opts) {
     // implicitStrip should be true by default
   this.implicitStrip = opts.implicitStrip === undefined ? true : Boolean(opts.implicitStrip);
-  this.abspath = String(opts.abspath) === opts.abspath ? path.resolve(opts.abspath) : '';
+  this.abspath = (String(opts.abspath) === opts.abspath && (opts.abspath!==null && String(opts.abspath).trim()!=='')) ? path.resolve(opts.abspath) : '';
```

Comment:
The `opts.abspath !== null` is redundant, as `String(x) === x` if and only if x is a string.

Put spaces between the equality check: `.trim() !== ''`